ui/app/components/ui/button-group/index.scss

```diff
@@ -4,7 +4,8 @@
   align-items: center;
 
   &__button {
-    font-size: 1rem;
+    @include H5;
```

Comment:
Was this issue resolved? https://github.com/MetaMask/metamask-extension/pull/9074/files#r465984022

Reply:
I don't think so!

Reply:
Fixed in the latest commit